modules/tinymce/src/themes/silver/test/ts/browser/editor/color/ColorPickerSanityTest.ts

```diff
@@ -97,6 +97,12 @@ describe('browser.tinymce.themes.silver.editor.color.ColorPickerSanityTest', ()
 
       it('TBA: Open dialog, pick a different color, click Cancel and assert color does not change', async () => {
```

Comment:
This needs updating to reflect the changes in the test
```suggestion
      it('TBA: Open dialog, pick black, reopen the dialog, pick a different color, click Cancel and assert color does not change', async () => {
```